src/modes/top10/Top10.svelte

```diff
@@ -223,16 +223,19 @@
     highlightTimeValue = value;
   }, 10);
 
-  function onHighlight(e) {
+  function resolveHighlightedTimeValue(e) {
```

Comment:
can you reuse the resolve function from the other file by importing it?

Reply:
Works.